sorting/c-or-cpp/radix-sort.cpp

```diff
@@ -0,0 +1,54 @@
+//Author: Tawfik Yasser
+#include <iostream>
+#include<algorithm>
+using namespace std;
+int getMax(int array[], int n) {
+  int max = array[0];
+  for (int i = 1; i < n; i++)
+    if (array[i] > max)
+      max = array[i];
+  return max;
+}
+void countingSort(int array[], int size, int place) {
+  const int max = 10;
+  int output[size];
+  int count[max];
+
+  for (int i = 0; i < max; ++i)
+    count[i] = 0;
+
+  for (int i = 0; i < size; i++)
+    count[(array[i] / place) % 10]++;
+
+  for (int i = 1; i < max; i++)
+    count[i] += count[i - 1];
+
+  for (int i = size - 1; i >= 0; i--) {
+    output[count[(array[i] / place) % 10] - 1] = array[i];
+    count[(array[i] / place) % 10]--;
+  }
+
+  for (int i = 0; i < size; i++)
+    array[i] = output[i];
+}
+
+void radixsort(int array[], int size) {
```

Comment:
typo : -
you are calling `radixSort()` and defined `redixsort()` here ( 'S' capital in function call )

Reply:
Sorry for that. I will edit it now